plugins/kubernetes/app/models/kubernetes/deploy_group_role.rb

```diff
@@ -93,6 +93,10 @@ def self.seed!(stage)
       end
     end
 
+    def self.istio_injection_supported?
+      Samson::EnvCheck.set?("ISTIO_INJECTION_SUPPORTED")
```

Comment:
`ISTIO_ENABLE_INJECTION` ?

Reply:
noun-action is usually nicer especially when we plan to nest more things .. `ISTIO_INJECTION_ENABLED` ... but "supported" also makes sense since users still have to opt-in

Reply:
I was following the convention of the `KRITIS_BREAKGLASS_SUPPORTED` env var.